src/components/pages/dashboard/DashboardTopArea.tsx

```diff
@@ -21,11 +22,11 @@ export const DashboardTopArea = ({ done, proc_start, proc_end }: OwnProps) => {
           </div>
           <div>
             <div>Started</div>
-            <div>{proc_start !== null ? proc_start : '-'}</div>
+            <div>{proc_start !== null ? getFormattedDate(proc_start) + getDistanteToNow(proc_start) : '-'}</div>
```

Comment:
Just displaying the different formattings next to each other. When applying styles I will separate these dates into different containers.